_info/Black_market

```diff
@@ -1,3 +1,5 @@
 Black_Market
 
-Not to be mistaken for the Market or the Stock market, this market holds a Varity of useful items for anyone to use, which can drastically change the situation. Note that the black market is only visible to the people in the respawning phase, and all the items purchased there can only be used when in the respawning phase. All the other conditions of the market applies to the black market as well.
+Not to be mistaken for the Market or the Stock market, this market holds a Varity of useful items for anyone to use, which can drastically change the situation. 
+Note that the black market is only visible to the people in the respawning phase, and all the items purchased there can only be used when in the respawning phase. 
+All the other conditions of the market applies to the black market as well. Use `!market view` to view the black market.
```

Comment:
i think it would help to specify that the commands are the same as regular market

Reply:
alright will fix